repose-aggregator/artifacts/src/config/filters/log4j2.xml

```diff
@@ -30,6 +30,7 @@
         <Logger name="org.springframework" level="warn"/>
         <Logger name="intrafilter-logging" level="info"/>
         <Logger name="org.openrepose.commons.utils.servlet.http.HttpServletResponseWrapper.headerWarning" level="warn"/>
+        <Logger name="trace-id-logging" level="trace"/>
```

Comment:
Looking at this again, should I set this to `info`? That is the current default for the `PowerFilter` logger, so I'm thinking probably so.